src/Output/Color.php

```diff
@@ -53,47 +53,26 @@ class Color
     protected string $format = "\033[:mod:;:fg:;:bg:m:txt:\033[0m";
 
     /** @var array Custom styles */
-    protected static array $styles = [];
-
-    /**
-     * Returns a line formatted as comment.
-     */
-    public function comment(string $text, array $style = []): string
```

Comment:
we should keep all these factory methods :)

Reply:
Updated!